lib/vsc/install/shared_setup.py

```diff
@@ -1158,11 +1158,6 @@ def cleanup(self, prefix=''):
                 except OSError:
                     log.error("cleanup failed for %s" % d)
 
-        for fn in ('setup.cfg',):
-            ffn = prefix + fn
-            if os.path.isfile(ffn):
-                os.remove(ffn)
```

Comment:
It was added in the initial commit...

Turns out when you create your own setup.cfg and tell vsc_install to not create one (using createsetupcfg=False) it will not create one, but first delete the one you created in this cleanup, before going to the create_rpm step, which is not what you want.

If you let it create one it will create one and overwrite the one that was there anyway, so no need to remove it first.